src/components/plots/linePlot.ts

```diff
@@ -41,7 +51,9 @@ export module Plot {
     }
 
     protected _getDrawer(key: string) {
-      return new Plottable._Drawer.Line(key);
+      var lineDrawer = new Plottable._Drawer.Line(key);
+      lineDrawer.interpolationMode(this._interpolationMode);
+      return lineDrawer;
```

Comment:
Not a huge deal, but a cleaner usage
return new Plottable._Drawer.Line(key).interpolationMode(this._interpolationMode);